src/integers/operator_precedence.md

```diff
@@ -18,11 +19,12 @@ None of this should be a surprise if you learned [PEMDAS](https://www.khanacadem
 // 6 + 27 / 2 - 2
 
 // 27 / 2 happens next
-// because of int division, that gives 13
+// because of int division, that gives 13 
+// (ignores float point number .5)
```

Comment:
I don't like this parenthetical line. 
* its `floating point` not `float point`
* The `int division` line above is already explaining that.

We should say `integer division` though, since that is what we call it in previous sections